src/components/node-list/node-list-group.js

```diff
@@ -20,7 +20,9 @@ export const NodeListGroup = ({
             <NodeListRow
               checked={!type.disabled}
               id={type.id}
-              label={type.name}
+              label={`${type.name} <i>(${type.nodeCount.visible}/${
```

Comment:
Is there a way `label` could optionally take a render function and this be in JSX?

Reply:
@bru5 Yeah I absolutely could, but this is [importing a utility from Kedro-UI](https://github.com/quantumblacklabs/kedro-ui/blob/master/src/components/search-results/search-results-utils.js), which functions this way. I don't remember why it does it this way, because I thought it used to use react components and Array.reduce. I could change it in Kedro-UI, but for now I'd rather keep it as is, and look into changing that in a future release?

The alternative is to stop importing the functionality from Kedro-UI, and replicate it here but modify it so that it generates an object instead. Reckon it's necessary?

Reply:
I see thanks for clarifying, sure that seems fair then to change it upstream when it becomes feasible later on.